README.rst

```diff
@@ -43,12 +43,18 @@ request to the main repository.
 Running the test suite
 =========================
 
-To run the test suite, you need the nose and coverage modules.
+To run the test suite, you need the pytest, pytest-cov and coverage modules.
```

Comment:
Technically not accurate, you only need pytest.

This is a pity that we can not have the same setup as we had with nose (i.e. always try to compute coverage and just warn if coverage is not installed). At the same time I don't think anyone was using coverage much locally.